### PR TITLE
Slaierno personal page

### PR DESCRIPTION
**Pull request by slaierno**

### CODE REVIEW

File: _authors/en/slaierno.md [hunk 1]
```diff
@@ -6,7 +6,19 @@ lang: en
 lang-ref: slaierno
 ---
 
-This is my personal page.
+Born in 1990, raised as a [*terrone*](https://en.wikipedia.org/wiki/Terrone) and [bolognese](https://en.wikipedia.org/wiki/Bologna) by choice. I am one of those kids that always had a computer to play with.
 
-⚠ _Work in progress._ ⚠
+My house first computer was a [Commodore Amiga 500](https://en.wikipedia.org/wiki/Amiga_500). There is a picture of when I was a year and half old where I am on my father laps while he was playing a videogame. Another picture, a year later, I was next to him, with the player 2 joystick controlling a jeep a giving support to my father's helicopter in [Silkworm](https://en.wikipedia.org/wiki/Silkworm_\(video_game\)).
```
Comment (kmfrick) [inline _authors/en/slaierno.md:11]: father's lap

File: _authors/slaierno.md [hunk 2]
```diff
@@ -70,7 +70,19 @@ lang: it
 lang-ref: slaierno
 ---
 
-Questa è la mia pagina personale.
+Classe 1990, cresciuto terrone e bolognese d'adozione. Sono uno di quelli che col computer in mano ci è nato.
 
-⚠ _Lavori in corso._ ⚠
+In casa avevamo un [Commodore Amiga 500](https://en.wikipedia.org/wiki/Amiga_500). In una foto in cui avevo un anno e mezzo sono ritratto sulle sue gambe mentre lui gioca. In un'altra foto dell'anno successivo ero accanto a lui con il secondo joystick in mano a dare supporto al [suo elicottero metnre ero a bordo di una jeep](https://en.wikipedia.org/wiki/Silkworm_\(video_game\)).
```
Comment (kmfrick) [inline _authors/slaierno.md:75]: mentre

File: _authors/slaierno.md [hunk 3]
```diff
@@ -70,7 +70,19 @@ lang: it
...
 
+Poco prima che iniziassi la scuola, mio padre comprò finalmente un PC. "Per aiutarmi nello studio" disse a mia madre. [Giochi per il Mio Computer](https://it.wikipedia.org/wiki/Giochi_per_il_mio_computer) era considerata un encicopledia, suppongo. E [Tomb Raider](https://en.wikipedia.org/wiki/Tomb_Raider_\(1996_video_game\)) serviva per aiutarmi con i compiti di storia, non c'è dubbio.
+
+Oltre ai videogiochi, l'intero mondo informatico mi ha sempre affascinato. Inizialmente giocavo al sistemista casalingo, poi durante il liceo mi avvicinai al mondo della programmazione e decisi che era troppo divertente per abbandonarlo.
+
+Dopo una piatta parentesi salentina durante la laurea triennale, ho concluso gli studi di Ingegneria Informatica all'Università di Bologna, dove mi sono principalmente concentrato sulla computer vision, passando per ricerca operativa, intelligenza artificiale ed FPGA. Mi sono anche interessato alla teoria dei linguaggi di programmazinoe.
```
Comment (kmfrick) [inline _authors/slaierno.md:81]: programmazione

File: _authors/en/slaierno.md [hunk 4]
```diff
@@ -6,7 +6,19 @@ lang: en
 lang-ref: slaierno
 ---
 
-This is my personal page.
+Born in 1990, raised as a [*terrone*](https://en.wikipedia.org/wiki/Terrone) and [bolognese](https://en.wikipedia.org/wiki/Bologna) by choice. I am one of those kids that always had a computer to play with.
 
-⚠ _Work in progress._ ⚠
+My house first computer was a [Commodore Amiga 500](https://en.wikipedia.org/wiki/Amiga_500). There is a picture of when I was a year and half old where I am on my father's lap while he was playing a videogame. Another picture, a year later, I was next to him, with the player 2 joystick controlling a jeep a giving support to my father's helicopter in [Silkworm](https://en.wikipedia.org/wiki/Silkworm_\(video_game\)).
```
Comment (ste001) [inline _authors/en/slaierno.md:11]: `The first computer in my house` sounds a bit better.

`controlling a jeep and giving support` on the last phrase.

**Pull request merged**